functions/accept.js

```diff
@@ -34,11 +59,73 @@ module.exports.handler = async function(
     QueueUrl
   }).promise();
   responseBody.sqsResult = "SUCCESS " + MessageId;
-  console.timeEnd("acceptSQS");
 
-  console.timeEnd("accept");
+  return response(200, responseBody);
+};
+
+function response(statusCode, body, headers = {}) {
   return {
-    statusCode: responseCode,
-    body: JSON.stringify(responseBody)
+    statusCode,
+    headers: Object.assign({ "Content-Type": "application/json" }, headers),
+    body: JSON.stringify(body)
   };
-};
+}
+
+async function authenticate({
+  method = "POST",
+  path,
+  queryStringParameters,
+  host,
+  port,
+  authorization
+}) {
+  const params = queryStringParameters
+    ? new URLSearchParams(queryStringParameters).toString()
+    : "";
+  const request = {
+    method,
+    url: path + params,
```

Comment:
I think we need a `?` between the path and the params.

Reply:
Hmm, yeah, for some reason I thought URLSearchParams did that. But now that I look at it, we don't actually use search params here. So I think I'll just take that out for now